### ISSUE TITLE
Steps

### ISSUE THREAD

**Issue by roccoavallone** [sparkfun/Stranger_Things_Wall#1]
What's the thing you have attached to the arduino? did you use the same christmas light wiring that you got the bulbs from to run power to the LED's? If you could post the full list of parts needed that would be much appreciated. thank you

**Comment by sprive** [comment]
The parts list is in the README. Note it is not a Build of Materials (BOM) but it has convenient links to the Sparkfun LEDs.  

To see what resistors and optional capacitors etc you need, you should see some of the SparkFun or Adafruit "NeoPixel" tutorials. Really - get a simple tutorial working before you attempt this more complex example.

I too would love to see more detail, such as a closeup of the wiring (I'm not sure how they managed to run 4 wires, when it appears from the photo they are using the original 120v 2-conductor wire (perhaps they clear-taped two additional hookup wires, and we just can't see it from the photo/YouTube). You have to expect some rough edges in these examples.

**Comment by mhord** [comment]
Attached to the Arduino is a USB-Serial breakout board (IIRC, I used a prototype for a next-gen device that should be coming out in a few weeks; an FTDI Basic would work just as well).

As for the wiring, I cheated, a lot. I pulled the sockets off of the original wiring and slipped them over the LEDs, which I wired together with bits of USB cable.

This is a pretty involved project; I spent probably 16 or more hours just wiring up the LEDs and making the bulb/socket combo to slip over each one.
